.circleci/config.yml

```diff
@@ -1,7 +1,7 @@
 version: 2.1
 
 orbs:
-  hmpps: ministryofjustice/hmpps@6.1
+  hmpps: ministryofjustice/hmpps@6.2
```

Comment:
We've started to drop pinning the minor version for the orb

Reply:
@andrewrlee would it be worth making that change to the Kotlin template?